android/titanium/src/java/org/appcelerator/titanium/view/TiGradientDrawable.java

```diff
@@ -74,21 +77,187 @@ public TiGradientDrawable(View view, KrollDict properties)
 			endPoint = new TiPoint((HashMap) endPointObject, 0, 1);
 		}
 
-		startRadius = TiConvert.toTiDimension(properties, "startRadius", TiDimension.TYPE_UNDEFINED);
-		if (startRadius == null) {
-			startRadius = DEFAULT_RADIUS;
+		// Fetch the start/end radius values for a "radial" gradient.
+		this.startRadius = TiConvert.toTiDimension(properties, "startRadius", TiDimension.TYPE_UNDEFINED);
```

Comment:
Minor, but here's a good opportunity to define `"startRadius"` and `"endRadius"` as constants